src/config/dune

```diff
@@ -22,5 +22,5 @@ let () =
      \  (targets config.mlh)\n\
      \  (mode fallback)\n\
      \  (deps %{profile}.mlh " ^ config_deps
-     ^ " (env_var CODA_DUNE_PROFILE))\n
+     ^ " (env_var MINA_DUNE_PROFILE))\n
```

Comment:
The value of this environment variable seems never to be used. Can we just delete it here?

Reply:
I think the idea is that if its value changes, the target is rebuilt:

  https://dune.readthedocs.io/en/stable/concepts.html?highlight=env_var#dependency-specification

So there need not be a mention of the variable anywhere else in the code for it to be used.

Reply:
I'm supposing the variable is meant for use in individual developer scripts.

Reply:
I don't think anyone has ever used it for this. I added it in #2303 by accident, not noticing that I had left it. My original version of that PR used environment variables before I realised we could use `%{profile}`.
Let's kill it.